agent/fw_drupal8.c

```diff
@@ -20,6 +21,60 @@
 
 #define PHP_PACKAGE_NAME "drupal/core"
 
+NR_PHP_WRAPPER(nr_drupal_exception) {
+  int priority = nr_php_error_get_priority(E_ERROR);
+  zval* event = NULL;
+  zval* exception = NULL;
+
+  /* Warning avoidance */
+  (void)wraprec;
+
+  NR_PHP_WRAPPER_REQUIRE_FRAMEWORK(NR_FW_DRUPAL8);
```

Comment:
Should this use a variation of the `NR_PHP_WRAPPER_REQUIRE_FRAMEWORK` macro (which doesnt yet exist) to make sure a compatible version of Drupal is present?  This works with 9+ correct?

Reply:
This fixes exceptions for 8 and exceptions _and_ errors for 9+. For Drupal 8 errors, this code has no impact (positive or negative) as the problem with error reporting originates from this [block](https://github.com/newrelic/newrelic-php-agent/blob/R11.4/agent/php_execute.c#L1389) unique to pre-oapi instrumentation, and D8 is only compatible with PHP up to 7.4.